.github/workflows/e2e-tests-preview.yml

```
@@ -0,0 +1,316 @@
+      - name: 📥 Checkout
+        uses: actions/checkout@v4
+
+      - name: 📊 Collect Test Results
+        uses: ./.github/actions/collect-test-results
+        with:
+          test-type: "e2e"
+          artifact-pattern: "e2e-results-*"
+
+      - name: 💬 Post Results to PR
+        if: github.event_name == 'pull_request'
+        uses: ./.github/actions/post-test-comment
```

actionlint (1.7.7)

missing input "commit-sha" which is required by action "Post Test Status Comment" defined at "./.github/actions/post-test-comment". all required inputs are "commit-sha", "github-token", "workflow-run-id"

(action)